src/core_plugins/console/api_server/spec/index.js

```diff
@@ -25,4 +26,17 @@ export function getSpec() {
 
     return { ...acc, ...spec };
   }, {});
+  extensionSpecFilePaths.forEach((extensionSpecFilePath) => {
+    const extensionFiles = glob.sync(join(extensionSpecFilePath, '*.json'));
+    extensionFiles.forEach((extensionFile) => {
+      merge(result, JSON.parse(readFileSync(extensionFile, 'utf8')));
```

Comment:
Just curious, if the spec file isn't valid JSON, what error is shown? If that error is not helpful, we might want to try/catch that and throw a more specific error

Reply:
No error is shown to the user, but there is an error in the Kibana logs about invalid json.